target/linux/bcm63xx/base-files/etc/board.d/02_network

```diff
@@ -21,7 +21,11 @@ t-com,speedport-w-500v)
 adb,a4001n1|\
 adb,a4001n|\
 adb,pdg-a4001n-a-000-1a1-ax|\
-adb,pdg-a4101n-a-000-1a1-ae|\
+adb,pdg-a4101n-a-000-1a1-ae)
```

Comment:
Here you are also changing the network configuration of adb,a4001n1, adb,a4001n, adb,pdg-a4001n-a-000-1a1-ax... and we don't want to do that.

Reply:
I fixed the problem. Sorry